docs/config.rst

```diff
@@ -214,6 +258,34 @@ The following configuration shows a bunch of mapping examples:
             </doctrine_mongodb:config>
         </container>
 
+    .. code-block:: php
+
+        use Symfony\Config\DoctrineMongodbConfig;
+
+        return static function (DoctrineMongodbConfig $config): void {
+            $config->documentManager('default')
+                ->mapping('MyBundle1')
+                ->mapping('MyBundle2')
+                    ->type('xml')
+                ->mapping('MyBundle3')
+                    ->type('attribute')
+                    ->dir(__DIR__.'/../src/Document')
+                ->mapping('MyBundle4')
+                    ->type('xml')
+                    ->dir(__DIR__.'/../config/doctrine/mapping')
```

Comment:
This is missing Resources. Or if `Resources` is no longer encouraged we should update previous examples